vassal-app/src/main/java/VASSAL/launch/EditorWindow.java

```diff
@@ -423,7 +423,7 @@ public void clearListKeyCommands() {
   boolean isTempFile(String name) {
     return name == null ||
       name.isEmpty() ||
-      ("tmp".equals(name.substring(0, 3)) && name.contains(".zip"));  //NON-NLS
```

Comment:
Is there some reason not to use `name.startsWith("tmp")` here?

Reply:
True. Updated.